modules/axruntime/src/signal.rs

```diff
@@ -0,0 +1,161 @@
+/* Copyright (c) [2023] [Syswonder Community]
+ *   [Rukos] is licensed under Mulan PSL v2.
+ *   You can use this software according to the terms and conditions of the Mulan PSL v2.
+ *   You may obtain a copy of Mulan PSL v2 at:
+ *               http://license.coscl.org.cn/MulanPSL2
+ *   THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT, MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ *   See the Mulan PSL v2 for more details.
+ */
+
+#[cfg(feature = "irq")]
+use core::sync::atomic::AtomicI64;
+use core::{ffi::c_int, time::Duration};
+
+#[allow(non_camel_case_types)]
+#[derive(Copy, Clone, Debug)]
+struct itimerval {
```

Comment:
These C structures should be put into `ctypes`

Reply:
I removed the definition of itimerval. Other structures are used in kernel and are started with `k_`.